app/controllers/vendor_api/applications_controller.rb

```diff
@@ -1,20 +1,22 @@
 module VendorAPI
   class ApplicationsController < VendorAPIController
+    VERSION = '1.0'.freeze
```

Comment:
Is this meant to be the base or introductory version for the set of endpoints served by the controller(s)? I'd consider keeping this information in the same place as the configuration which ties the presentation concerns to the version if possible.
What happens in version `2.0` for presumably ubiquitous controllers like `ApplicationsController` ?